### PR TITLE
Add grid CSS and basic layout of contact page

### PR DESCRIPTION
**Pull request by buzuloiu**
Issue: #5 

image

### CODE REVIEW

**Comment by buzuloiu** [comment]
This still needs the proper content added to it and some of the styling fixed, plus the donate button doesn't link to anything yet, but it's a start.

**Comment by OldToast** [comment]
The donation button should be split into two buttons, one for gofundme and one for tax receipt donations.

There should also be instructions on how to donate to get a tax receipt.

**Comment by azqlin** [comment]
Use `<p align="justify"></p>` instead of `<center></center>`, so it look nicer :D

**Comment by azqlin** [comment]
Maybe we can combine our column classes in the CSS file as I have `<div class="column2">` and `<div class="column3">` too.

**Comment by azqlin** [comment]
Now looks like this:
image

File: index.html [hunk 1]
```diff
@@ -87,21 +90,55 @@ <h1>Rocketry</h1>
...
+                                        <p align="justify">
+                                            To achieve our goals, we need your help. Your contribution will directly support our goals of being out of this world.
+                                        </p>
+                                        <p align="justify">
+                                            Your donation helps create a collaborative environment where students are able to apply their studies out of the lecture hall, meet, and compete against students from around the globe!
+                                        </p>
+                                        <p align="center" style="margin-top:0">
+                                            <font size="3"><i>1.5x team match from uOttawa for any donations made before Dec 1!</i></font>
+                                        </p>
+                                        <form method="post" target="_blank" action="https://www.gofundme.com/f/uottawa-rocketry?fbclid=IwAR0bVBM4bsLN6Z4ULkP6KnYIHRyVpGvFN4Qgi9Crl2GxmiJjAUKpwtlC2xc">
```
Comment (buzuloiu) [inline index.html]:
```suggestion
                                        <form method="post" target="_blank" action="https://www.gofundme.com/f/uottawa-rocketry">
```

**Review by buzuloiu — COMMENTED** [review]
I don't think you need this facebook tracking ID, but LGTM other than that! 🚢

**Pull request merged**